src/GlobalHeader/stories/GlobalHeader.stories.js

```diff
@@ -1 +1,254 @@
+              path: 'https://adobe.io'
+            },
+            {
+              title: 'Document Cloud View SDK',
+              path: 'https://adobe.io'
+            },
+            {
+              title: 'Adobe Acrobat',
+              path: 'https://adobe.io'
+            },
+            {
+              title: 'Adobe Target',
+              path: 'https://adobe.io'
+            },
+            {
+              title: 'Adobe I/O Runtime',
+              path: 'https://adobe.io'
+            },
+            {
+              title: 'Project Firefly',
+              path: 'https://adobe.io'
+            },
+            {
+              title: 'Adobe I/O Events',
+              path: 'https://adobe.io'
+            }
+          ]
+        },
+        {
+          pages: [
+            {
+              title: 'Adobe XD',
+              path: 'https://adobe.io'
+            },
+            {
+              title: 'Adobe Photoshop',
+              path: 'https://adobe.io'
+            },
+            {
+              title: 'Document Cloud View SDK',
+              path: 'https://adobe.io'
+            },
+            {
+              title: 'Adobe Acrobat',
+              path: 'https://adobe.io'
+            },
+            {
+              title: 'Adobe Target',
+              path: 'https://adobe.io'
+            },
+            {
+              title: 'Adobe I/O Runtime',
+              path: 'https://adobe.io'
+            },
+            {
+              title: 'Project Firefly',
+              path: 'https://adobe.io'
+            },
+            {
+              title: 'Adobe I/O Events',
+              path: 'https://adobe.io'
+            }
+          ]
+        }
+      ]
+    }
+  ]
+}
+
+const pages = [
+  {
+    title: 'Home',
+    path: '/'
+  },
+  {
+    title: 'Guides',
+    path: '/guides/'
+  }
+]
+
+const location = {
+  pathname: '/'
+}
+
+export const Default = () => {
+  const globalNavCopy = { ...globalNav }
+  globalNavCopy.menus = [globalNavCopy.menus[1]]
+
+  return (
+    <View position='fixed' height='size-800' left='0' right='0'>
+      <GlobalHeader
+        location={location}
+        pages={pages}
+        globalNav={globalNavCopy}
+      />
+    </View>
+  )
+}
+
+export const WithMultipleMenus = () => {
+  return (
+    <View position='fixed' height='size-800' left='0' right='0'>
+      <GlobalHeader location={location} pages={pages} globalNav={globalNav} />
+    </View>
+  )
+}
+
+export const WithDocs = () => {
+  const globalNavCopy = { ...globalNav }
+  globalNavCopy.menus = [globalNavCopy.menus[0]]
+
+  const docs = {
+    path: '/docs/'
+  }
+
+  return (
+    <View position='fixed' height='size-800' left='0' right='0'>
+      <GlobalHeader
+        docs={docs}
+        location={location}
+        pages={pages}
+        globalNav={globalNavCopy}
+      />
+    </View>
+  )
+}
+
+export const WithConsoleAndSignIn = () => {
+  globalNav.console = true
+  globalNav.signIn = true
+
```

Comment:
Need to set:

```
  globalNav.console = false
  globalNav.signIn = false
```

in all the other stories or else if you view this story you can't make but buttons go away without rebooting.